war/src/main/js/add-item.js

```diff
@@ -246,15 +245,23 @@ $.when(getItems()).done(function(data) {
             showInputHelp('.add-item-name');
             setFieldValidationStatus('name', true);
             if (getFormValidationStatus()) {
-              enableSubmit(true);
+              if (event.type === 'blur') { 
+                enableSubmit(true);
+              } else if (event.type === 'keyup') {
+                var btn = $('form .footer .btn-decorator button[type=submit]');
```

Comment:
The btn variable defined again since it was defined only locally before and needed again now.